File: Assets/scenes/warzonescene/script/WaypointBehaviour.js
#pragma strict

private var wpController : WaypointController;

function Start () {
	wpController = GameObject.Find("SceneScripts").GetComponent.<WaypointController>();
}

function Update () {

}

function OnTriggerEnter(col : Collider){
	if(col.name == "TheShip"){
		wpController.gateDone(gameObject);
	}
}

File: Assets/scenes/warzonescene/script/WaypointBehaviour.cs
using UnityEngine;
using System;


public class WaypointBehaviour:MonoBehaviour{
	
	WaypointController wpController;
	public int id = 0;
	
	public void Start() {
	}
	
	public void Update() {
	
	}

	public void setController(WaypointController wc){
		wpController = wc;
	}
	
	public void OnTriggerEnter(Collider col){
		if(col.name == "TheShip"){
			wpController.gateDone(gameObject);
		}
	}
}

